src/binder/expression_binder.cpp

```diff
@@ -121,5 +122,13 @@ void validateAggregationExpressionIsNotNested(const Expression& expression) {
     }
 }
 
+std::string ExpressionBinder::getUniqueName(const std::string& name) const {
+    return binder->getUniqueExpressionName(name);
+}
+
+main::ClientContext* ExpressionBinder::getClientContext() {
```

Comment:
Anyone using this function? I don't think generally this is the right way to get client context.